observermedia/eight-day-week
# Feature: Include images & captions into export

### Description of the Change

New feature to include images & captions in the export file, see https://github.com/10up/eight-day-week/issues/22

Closes #22

### How to test the Change
1. Once there are some blog posts with images & captions, go to Print > Print Issues
2. Under "Export for InDesign, click "Export All"
3. Unzip the archive and view the .xml file to ensure the images & captions where exported with the rest of the content

### Changelog Entry

> Added - New feature to include images & captions in the export

### Credits
Props @bmarshall511 

### Checklist:
